src/Library/IPCServerPrivate.cpp

```diff
@@ -84,6 +84,8 @@ namespace usbguard
       IPCServer::AccessControl::Privilege::MODIFY);
     registerHandler<IPC::getParameter>(&IPCServerPrivate::handleGetParameter, IPCServer::AccessControl::Section::PARAMETERS,
       IPCServer::AccessControl::Privilege::LIST);
+    registerHandler<IPC::checkIPCPermissions>(&IPCServerPrivate::handleCheckIPCPermissions, IPCServer::AccessControl::Section::ALL,
+      IPCServer::AccessControl::Privilege::NONE);
```

Comment:
``IPCServer::AccessControl::Section::ALL, IPCServer::AccessControl::Privilege::NONE``
Is this the right approach?

Reply:
Makes sense imo.